silk-core/src/main/kotlin/net/silkmc/silk/core/event/ServerEvents.kt

```diff
@@ -0,0 +1,36 @@
+package net.silkmc.silk.core.event
+
+import net.minecraft.server.MinecraftServer
+import net.silkmc.silk.core.annotations.ExperimentalSilkApi
+
+/**
+ * Events related to the [MinecraftServer].
+ */
+@Suppress("unused") // receiver is for namespacing only
+val Events.server get() = ServerEvents
+
+@ExperimentalSilkApi
+object ServerEvents {
+
+    class ServerEvent(val server: MinecraftServer)
+
+    /**
+     * Called before the server is initialized.
+     */
+    val preStart = Event.bothImmutable<ServerEvent>()
+
+    /**
+     * Called after the server has successfully started.
+     */
+    val started = Event.bothImmutable<ServerEvent>()
```

Comment:
I kinda think it'd be more consistent to have this named `postStart`

Reply:
I decided to go with your suggestion and named them `postStart` and `postStop`.